tests/TestCase/View/Helper/PlaceholdersHelperTest.php

```diff
@@ -152,7 +152,7 @@ public function testGetTemplate()
      */
     public function testDefaultTemplater()
     {
-        $contents = $this->Placeholders::defaultTemplater($this->object, 'body', [$this->image1, $this->image2], fn ($entity, $params) => sprintf('%s %s', $entity->title, $params ? json_encode($params) : ''));
+        $contents = $this->Placeholders::defaultTemplater($this->object, 'body', [$this->image1, $this->image2], fn ($entity, $params) => sprintf('%s %s', $entity->title, $params ? $params : ''));
```

Comment:
Hmmm… this change means that the `defaultTemplater` has changed its behavior in regards to the format of the `$params`, i.e. the second argument passed to the callback function.

Reply:
No, the behavior should be the same. I think we were wrong in the data mock above to use an array as if the parameter arrived already transformed from string to array.

Reply:
Confirmed, you're right.